### DISCUSSION TITLE
How do I use ARCS cheat menu? I already now how to use wills cheat menu but not arcs cheat menu

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#905] [(C) Suggestions]
so anyways anyone who wants to create stuff come here!!!

**Comment by ghost** [comment]
1. `https://raw.githubusercontent.com/PatheticMustan/ProdigyMathGameHacking/HEAD/willsCheatMenu/dist/bundle.js`

**Reply by ghost** [reply]
hi

**Reply by LeoBadeaux** [reply]
Bonjour, @BETPRODIGYHACKER. Avoir une bonne journée? Aussi, arrêtez de vous faire passer pour PatheticMustan et changez votre photo de profil.

**Reply by NX721** [reply]
French

**Comment by ArcerionDev** [comment]
my cheat menu died, I never maintained it and I decided to just start working on will's menu instead

**Reply by ghost** [reply]
*gasp* sad

**Reply by NX721** [reply]
lmao
